app/views/main_page/landing.html.haml

```diff
@@ -0,0 +1,15 @@
+%body.landing-body
+  .landing-content
+    %a.title-logo{:href => root_path}
+      %span.first-word Canteen
+      %span.second-word Koala
+    %h3 Enjoy delicious food from the finest restaurants.
+    - if logged_in?
+      %h1 Hello #{ @current_user.name }
+    .button.explore-btn= link_to 'Explore Products', main_page_home_path
```

Comment:
it's a good practice to group the logic by using new lines, it will make the code easier to read :-)